cmake/BLTInstallableMacros.cmake

```diff
@@ -428,37 +428,161 @@ endmacro(blt_add_target_compile_flags)
 
 
 ##------------------------------------------------------------------------------
-## blt_convert_to_system_includes(TARGET <target>)
+## blt_find_target_dependencies(TARGET <target> TLIST <list name>)
 ##
-## Converts existing interface includes to system interface includes.
+## Store all target's dependencies (link libraries and interface link libraries)
+## recursively in the variable name TLIST holds.
 ##------------------------------------------------------------------------------
```

Comment:
Did anything in `blt_find_target_dependencies` change? Or was it just moved to a new file?